content/millennium/dstu2/individuals/patient.md

```diff
@@ -71,11 +75,28 @@ Notes:
 - The `birthdate` parameter may be provided twice to indicate a date range, but must contain the inclusive prefixes 'le' and 'ge'
 - The `birthdate` parameter may be provided once with the following prefixes: 'ge', 'le', 'gt', 'lt', 'eq'
 
-### Response
+### Headers
```

Comment:
I made the existing headers method have a default case for this so that we're not copying the headers around. We have a lot of common text like this that we don't just generate, and it is something that starting from the template would help with. I'm not sure the text actually provides much... I could also make the default headers link back to the media type and auth sections on the overview, and remove the text intro all together...